tests/LimitOrderTest.php

```diff
@@ -46,7 +46,6 @@ public function testReverse()
         $ii->addExpression('total_gross', '[total_net]+[total_vat]');
         $ii->getField($ii->id_field)->system = false;
 
-        // pass parameters as CSV string
```

Comment:
this line belongs to CSV in setOrder

Reply:
True, but very unclear comment, comments like this are bad - code change a comments very often do not.